### DISCUSSION TITLE
Audio lags video after pause & video turnes a bit reddish after pause or sometimes while playing

### DISCUSSION THREAD

**Discussion by butchnn** [smplayer-dev/smplayer#1108] [General]
Hi all.
I have 2 issues in this one question.
1st, when I pause & then restart the audio lags the video.
I can’t understand why.
2nd, a lot if times after a pause the entire video shifts to a slightly reddish tint/hue imbalance as if everyone looks a tad sunburned & even the background is slightly redder.
Sometimes it just happens on its own.
I hope I explained that one correctly.

My laptop is an Alienware M18 R1
i9 13900HX CPU
GeForce RTX 4090 GPU
Crucial DDR5 64GB RAM
Dual NVMe WD_BLACK SN850X 4GB SSD’s

This is my SMPlayer  version info:
Version: 24.5.0 (revision 10277) (64-bit)
Using Qt 5.15.2 (compiled with Qt 5.15.2)
Using MPV v0.37.0-522-ga56d8ff1

Anyway, does anyone know why these issues happen & how to correct them?
Thanks

**Comment by smplayer-dev** [comment]
I don't have this problem in my computer. Just in case try selecting another video and/or audio output in preferences -> general.
